scripts/demo-projects.android.sh

```diff
@@ -31,22 +30,24 @@ if [ "$REACT_NATIVE_COMPAT_TEST" = "true" ]; then
 fi
 
 pushd examples/demo-react-native
-run_f "npm run test:android-release-ci"
-DETOX_EXPOSE_GLOBALS=0 run_f "npm run test:android-release-ci"
+  run_f "npm run test:android-release-ci"
+
+  # Run tests with bloated JS bundle:
+  source $(dirname "$0")/demo-rn-bloat-bundle-test.sh
```

Comment:
$(dirname "$0") occurs a couple of times in this file - should probably be assigned to a descriptive var name to make it clearer